tools/utils.go

```
@@ -59,3 +59,23 @@ func getDescription(sc *bufio.Scanner, idx int, endLine string) (string, int, er
 	}
 	return description, idx, nil
 }
+
+func getDescriptionConnections(sc *bufio.Scanner, idx int, endLine string) (string, int, error) {
+	description := rightofEquals(sc.Text())
+	nLines := 0
+	for sc.Scan() {
+		idx++
+		line := sc.Text()
+		if strings.HasPrefix(line, endLine) {
+			return description, idx, nil
+		}
+		if line != "" {
+			if nLines > 0 {
+				description += "\n"
+			}
+			description += line
+			nLines++
+		}
+	}
+	return description, idx, nil
```

Do you want to return an error here? If you scan through all lines of the file and never hits the end of the description, I would think that would be a problem.

---

Since we will be merging the two description functions, this issue no longer exists.